Microsoft.HealthVault.Fhir/Units/UnitResolver.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.HealthVault.Fhir.Units
+{
+    internal class UnitResolver
+    {
+        private static UnitResolver _instance;
+
+        public List<UnitConversion> UnitConversions { get; set; }
+
+        private UnitResolver()
+        {
+            UnitConversions = JsonConvert.DeserializeObject<List<UnitConversion>>(File.ReadAllText(@"Data\UnitConversions.json"));
+        }
+
+        public static UnitResolver Instance
```

Comment:
This has disadvantage of multiple threads instantiating the instance .. here are more details ... https://msdn.microsoft.com/en-us/library/ff650316.aspx